experiment/find-merge-conflicts.sh

```diff
@@ -65,23 +65,23 @@ while true; do
   done
 done
 
-if [[ -z "${conflicting_prs[@]:-}" ]]; then
+if [[ -z "${conflicting_prs[*]}" ]]; then
   echo "# No PRs with conflicts found." >&2
   exit
 fi
 
 echo "# ${#conflicting_prs[@]} conflicting PRs found" >&2
 
 tmpdir=$(mktemp -d)
-trap "rm -rf ${tmpdir}" EXIT
+trap 'rm -rf ${tmpdir}' EXIT
```

Comment:
Why do we want to evaluate `$tmpdir` at runtime?

Reply:
mostly because shellcheck complains